### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by jessepurcell**
Hey @Lucas-obrien
Can you please have a look through my prac for this week?

### CODE REVIEW

File: prac_08/convert_miles_km.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        try:
+            miles = float(self.root.ids.text_input.text)
+            self.conversion_result = str(miles * self.MILES_TO_KM_FACTOR)
+        except ValueError:
+            self.conversion_result = '0.0'
+
+    def handle_increment(self, amount):
+        """Increment the input_field by a passed in amount."""
+        input_field_text = self.root.ids.text_input.text
+        try:
```
Comment (Lucas-obrien) [inline prac_08/convert_miles_km.py:35]: Everything looks good here, my only suggestion would be considering making the error checking itself into a function, Lindsay's solution uses one for all handling of conversions :)

File: prac_08/dynamic_labels.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Estimated Time: 10 minutes
+Actual Time: 10 minutes
+"""
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+from kivy.uix.label import Label
+
+
+class SquareNumberApp(App):
```
Comment (Lucas-obrien) [inline prac_08/dynamic_labels.py]: Just need to fix this class name up

**Review by Lucas-obrien — CHANGES_REQUESTED** [review]
This week was a nice reprieve from the last! Just a couple things to have a look at!

File: prac_08/dynamic_labels.py (hunk 2 above)
Comment (jessepurcell) [reply prac_08/dynamic_labels.py]: Nice catch. Will do.

File: prac_08/convert_miles_km.py (hunk 1 above)
Comment (jessepurcell) [reply prac_08/convert_miles_km.py:35]: I'll look into it as it might simplify the code and improve readability, thanks. 👍

**Pull request merged**